src/DotLiquid/Strainer.cs

```diff
@@ -15,6 +15,27 @@ static class DictionaryExtensions
                 return dic[key] = factory();
             return found;
         }
+
+    }
+
+    static class MethodInfoExtensions

```

Comment:
specify the access explicitly always (public/protected/internal/private)

Reply:
Making these explicitly public helped me realize they need to be in their own files. I moved them to the Util namespace: https://github.com/dotliquid/dotliquid/pull/453/commits/89f65da331b408202dcae2200262a6eb670146f9